_posts/0000-01-02-astroshdud7.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
``
[introduction-to-github.pdf](https://github.com/astroshdud7/github-slideshow/files/3650753/introduction-to-github.pdf)

```